stack/generic-guide-topic-type.rst

```diff
@@ -0,0 +1,127 @@
+########################
+Generic guide topic type
+########################
+
+The generic guide topic type is the template to use when writing guides that are not covered by more specific topic types.
+
+.. note::
+
+   As topic types are introduced for how-to guides, tutorials, conceptual overviews, tasks, and so on, we should migrate topics written with the generic guide topic type to those more specific topic types.
+
+.. _generic-guide-topic-filename:
+
+File name and location
+======================
+
+Topics are reStructuredText files with an ``rst`` extension.
+
+If the topic is associated with a module, the topic’s file is located in the :ref:`module documentation directory <docdir-module-doc-directories>`.
+For example:
+
+.. code-block:: text
+
+   doc/lsst.example/my-topic.rst
+
+If the topic is associated with a package, the topic’s file is located in the :ref:`package documentation directory <docdir-package-doc-directory>`.
+For example:
+
+.. code-block:: text
+
+   doc/example/my-topic.rst
+
+In general, you shouldn’t create additional subdirectories to organize these files so that URLs and clean and simple.
+
+Use these guidelines for naming the file:
+
+- Name the file by adapting the title into a short phrase.
+
+- Connect words with a single hyphen (``-``) character.
+  You may only use periods (``.``) and underscores (``_``) when the file name includes an API that naturally contains periods and underscores.
+
+- Use lowercase, except for API names that naturally include uppercase characters.
+
+.. _generic-guide-topic-preamble:
+
+Preamble
+========
+
+If a topic is located in a :ref:`module documentation directory <docdir-module-doc-directories>`, you should add a ``py:currentmodule`` directive at the top of the file.
+For example, if the module’s name is ``lsst.example``, the directive is written as:
+
+.. code-block:: rst
+
+   .. py:currentmodule:: lsst.example
+
+This allows you to reference APIs relative to that base namespace.
+For example these two Python API cross-references are equivalent:
+
+.. code-block:: rst
+
+   `MyClass` and `lsst.example.MyClass`.
+
+If a topic is located in a :ref:`package documentation directory <docdir-package-doc-directory>`, you generally shouldn’t use the ``py:currentmodule`` directive.
+
+.. _generic-guide-topic-title:
+
+Title
+=====
+
+Title the topic according to these guidelines:
+
+- Use sentence case.
+  The first word is capitalized and following words are not, unless they are proper nouns or API names with capital letters.
+
+- Be descriptive so that a reader knows what to expect.
```

Comment:
Maybe say that the document and its title should be treated as standalone, not assuming that the user navigated to it in any particular way, with any particular context.